### ISSUE TITLE
[BUG] Deprecation error launching app debug mode.

### ISSUE THREAD

**Issue by t3nshi** [casinocoin/tokenwallet-mobile#19]
**Describe the bug**
In the side menu, when tap the logout button it does nothing

**To Reproduce**
1. Start the app
2. connected to debug console through dev tools
3. it shows warning:
```
[DEPRECATED][ion-menu] Using the [main] attribute is deprecated, please use the "contentId" property instead:
```

**Expected behavior**
Debug console should show debug information without deprecation errors

**Smartphone (please complete the following information):**
 - Device: Moto Z2 Force
 - OS: android 7
 - App Version  0.0.9

**Comment by t3nshi** [comment]
It will be fixed in my next PR

**Comment by t3nshi** [comment]
fixed on #20
